pages.es/linux/systemctl.md

```diff
@@ -11,26 +11,26 @@
 
 `systemctl --failed`
 
-- Inicia/Para/Reinicia/Recarga un servicio:
+- Inicia, para, reinicia, recarga o muestra el estado de una unidad:
 
-`systemctl {{start|stop|restart|reload}} {{unidad}}`
+`systemctl {{start|stop|restart|reload|status}} {{unidad}}`
 
-- Muestra el estado de una unidad:
+- Habilita o deshabilita una unidad para iniciarla al arrancar:
 
-`systemctl status {{unidad}}`
+`systemctl {{enable|disable}} {{unidad}}`
 
-- Habilita/Deshabilita una unidad para que se inicie en el arranque:
+- Reincia systemd, lee unidades nuevas o modificadas:
 
-`systemctl {{enable/disable}} {{unidad}}`
+`systemctl daemon-reload`
 
-- Enmascara/Desenmascara una unidad para evitar su habilitación y activación manual:
+- Checa si una unidad está activa, habilitada, o en estado fallido:
```

Comment:
Checa -> Verifica

Reply:
Why? They can be considered synonyms [^1]

[^1]: https://www.fundeu.es/consulta/checar-10200